eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/HttpCommand.java

```diff
@@ -57,9 +60,9 @@ public class HttpCommand implements ProtocolTransportObject {
 
     public String httpVersion;
 
-    public Header header;
+    @Getter @Setter private transient Header header;
```

Comment:
you do not need to add @Getter @Setter, because this class has the @Data annotation. @harshithasudhakar